wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -445,14 +445,6 @@ extension WireCallCenterV3 {
         }
     }
 
-    /// Call this method when the network quality of a participant changes and avs calls the `wcall_network_quality_h`.
```

Comment:
This was dead code right?

Reply:
yes, it updates the `networkQuality` for individual participants but we don't need it